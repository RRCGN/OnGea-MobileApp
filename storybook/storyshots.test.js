import initStoryshots from 'storyshots'
initStoryshots({
  configPath: './storybook/storybook-test.js'
})
