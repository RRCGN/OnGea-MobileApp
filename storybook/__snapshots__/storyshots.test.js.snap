exports[`Storyshots ButtonList Normal 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
          "marginLeft": -4,
          "marginRight": -4,
        },
        Object {
          "justifyContent": "flex-end",
        },
      ]
    }>
    <View
      style={
        Object {
          "marginLeft": 4,
          "marginRight": 4,
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <View>
          <View
            style={
              Array [
                Object {
                  "elevation": 2,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                },
                Object {
                  "borderRadius": 2,
                  "flex": -1,
                  "height": 36,
                  "justifyContent": "center",
                  "minWidth": 64,
                },
                Object {
                  "backgroundColor": "#00796B",
                },
                undefined,
              ]
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "bold",
                    "textAlign": "center",
                  },
                  Object {
                    "color": "white",
                  },
                ]
              }>
              BUTTON
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "marginLeft": 4,
          "marginRight": 4,
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}>
        <View>
          <View
            style={
              Array [
                Object {
                  "elevation": 2,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                },
                Object {
                  "borderRadius": 2,
                  "flex": -1,
                  "height": 36,
                  "justifyContent": "center",
                  "minWidth": 64,
                },
                Object {
                  "backgroundColor": "white",
                },
                undefined,
              ]
            }>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {
                    "fontSize": 14,
                    "fontWeight": "bold",
                    "textAlign": "center",
                  },
                  Object {
                    "color": "#00796B",
                  },
                ]
              }>
              BUTTON
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card Segment big 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <View
      style={
        Array [
          false,
          Object {
            "padding": 16,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card Segment big w/ border 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <View
      style={
        Array [
          Object {
            "borderBottomColor": "#E0E0E0",
            "borderBottomWidth": 1,
          },
          Object {
            "padding": 16,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
    <View
      style={
        Array [
          false,
          Object {
            "padding": 16,
          },
          undefined,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card Segment small 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <View
      style={
        Array [
          false,
          undefined,
          Object {
            "padding": 8,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card Segment small w/ border 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <View
      style={
        Array [
          Object {
            "borderBottomColor": "#E0E0E0",
            "borderBottomWidth": 1,
          },
          undefined,
          Object {
            "padding": 8,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
    <View
      style={
        Array [
          false,
          undefined,
          Object {
            "padding": 8,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card Title 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}>
      <View>
        <View
          style={
            Object {
              "aspectRatio": 1.7777777777777777,
            }
          }>
          <View
            style={
              Object {
                "flex": 1,
                "position": "relative",
              }
            }>
            <Image
              source={
                Object {
                  "uri": "http://placehold.it/1200x900",
                }
              }
              style={
                Object {
                  "alignSelf": "stretch",
                  "flex": 1,
                  "height": undefined,
                  "resizeMode": "cover",
                  "width": undefined,
                }
              } />
            <BVLinearGradient
              colors={
                Array [
                  0,
                  3422552064,
                ]
              }
              end={undefined}
              locations={null}
              start={undefined}
              style={
                Object {
                  "bottom": 0,
                  "left": 0,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "paddingTop": 24,
                  "position": "absolute",
                  "right": 0,
                }
              }>
              <Text
                accessible={true}
                allowFontScaling={true}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "backgroundColor": "transparent",
                      "color": "white",
                    },
                    Object {
                      "fontSize": 24,
                    },
                  ]
                }>
                Card Title
              </Text>
            </BVLinearGradient>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots Card Card View 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 20,
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": 2,
        "shadowOffset": Object {
          "height": 1,
        },
        "shadowOpacity": 0.18,
        "shadowRadius": 1.4,
      }
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      Lorem ipsum
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Alert Hint 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#F44336",
        },
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a mad message! I\'m mad.
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Alert Hint w/ icon 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#F44336",
        },
      ]
    }>
    <View
      style={
        Object {
          "height": 18,
          "marginRight": 12,
          "width": 18,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 18,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }>
        
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a mad message! I\'m mad.
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Nice Hint 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#8BC34A",
        },
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a friendly message.
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Nice Hint w/ icon 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#8BC34A",
        },
      ]
    }>
    <View
      style={
        Object {
          "height": 18,
          "marginRight": 12,
          "width": 18,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 18,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }>
        
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a friendly message.
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Touchable Hint 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View>
      <View
        style={
          Array [
            Object {
              "flex": 0,
              "flexDirection": "row",
              "padding": 12,
            },
            Object {
              "backgroundColor": "#8BC34A",
            },
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "flex": 1,
            }
          }>
          Press me!
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots Hint Warning Hint 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#FFC107",
        },
      ]
    }>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a slightly confused message.
    </Text>
  </View>
</View>
`;

exports[`Storyshots Hint Warning Hint w/ icon 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "flex": 0,
          "flexDirection": "row",
          "padding": 12,
        },
        Object {
          "backgroundColor": "#FFC107",
        },
      ]
    }>
    <View
      style={
        Object {
          "height": 18,
          "marginRight": 12,
          "width": 18,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 18,
            },
            undefined,
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }>
        
      </Text>
    </View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "flex": 1,
        }
      }>
      This is a slightly confused message.
    </Text>
  </View>
</View>
`;

exports[`Storyshots OGDate Date 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "rgba(0,0,0,0.54)",
            "fontSize": 12,
          },
          false,
        ]
      }>
      from
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "rgba(0,0,0,0.87)",
            "fontSize": 16,
            "fontWeight": "500",
          },
          false,
        ]
      }>
      10.04.2017
    </Text>
  </View>
</View>
`;

exports[`Storyshots OGDate Date List 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Object {
        "flexDirection": "row",
        "marginLeft": -16,
        "marginRight": -16,
      }
    }>
    <View
      style={
        Object {
          "paddingLeft": 16,
          "paddingRight": 16,
        }
      }>
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgba(0,0,0,0.54)",
                "fontSize": 12,
              },
              false,
            ]
          }>
          from
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgba(0,0,0,0.87)",
                "fontSize": 16,
                "fontWeight": "500",
              },
              false,
            ]
          }>
          10.04.2024
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "paddingLeft": 16,
          "paddingRight": 16,
        }
      }>
      <View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgba(0,0,0,0.54)",
                "fontSize": 12,
              },
              false,
            ]
          }>
          to
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "rgba(0,0,0,0.87)",
                "fontSize": 16,
                "fontWeight": "500",
              },
              false,
            ]
          }>
          16.04.2024
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots OGDate Light Date 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "rgba(0,0,0,0.54)",
            "fontSize": 12,
          },
          Object {
            "color": "white",
          },
        ]
      }>
      from
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "rgba(0,0,0,0.87)",
            "fontSize": 16,
            "fontWeight": "500",
          },
          Object {
            "color": "white",
          },
        ]
      }>
      10.04.2017
    </Text>
  </View>
</View>
`;

exports[`Storyshots OGIconButton Normal 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 18,
              "flex": 0,
              "height": 36,
              "justifyContent": "center",
              "width": 36,
            },
            undefined,
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#00796B",
                "fontSize": 18,
              },
              undefined,
              Object {
                "fontFamily": "Material Design Icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots OGTextButton Flat 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View>
      <View
        style={
          Array [
            Object {
              "paddingLeft": 8,
              "paddingRight": 8,
            },
            Object {
              "borderRadius": 2,
              "flex": -1,
              "height": 36,
              "justifyContent": "center",
              "minWidth": 64,
            },
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 14,
                "fontWeight": "bold",
                "textAlign": "center",
              },
              Object {
                "color": "#00796B",
              },
            ]
          }>
          BUTTON
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots OGTextButton Flat w/ custom color 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View>
      <View
        style={
          Array [
            Object {
              "paddingLeft": 8,
              "paddingRight": 8,
            },
            Object {
              "borderRadius": 2,
              "flex": -1,
              "height": 36,
              "justifyContent": "center",
              "minWidth": 64,
            },
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 14,
                "fontWeight": "bold",
                "textAlign": "center",
              },
              Object {
                "color": "red",
              },
            ]
          }>
          BUTTON
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots OGTextButton Raised 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}>
    <View>
      <View
        style={
          Array [
            Object {
              "elevation": 2,
              "paddingLeft": 16,
              "paddingRight": 16,
            },
            Object {
              "borderRadius": 2,
              "flex": -1,
              "height": 36,
              "justifyContent": "center",
              "minWidth": 64,
            },
            Object {
              "backgroundColor": "#00796B",
            },
            undefined,
          ]
        }>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "fontSize": 14,
                "fontWeight": "bold",
                "textAlign": "center",
              },
              Object {
                "color": "white",
              },
            ]
          }>
          BUTTON
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`Storyshots OGTravelDate Normal 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
      "padding": 30,
    }
  }>
  <View
    style={
      Object {
        "flex": 0,
        "flexDirection": "row",
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "flex-start",
          "height": 40,
          "justifyContent": "center",
          "marginRight": 16,
          "width": 40,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 24,
            },
            Object {
              "color": "rgba(0, 0, 0, 0.54)",
              "flex": 0,
            },
            Object {
              "fontFamily": "Material Design Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }>
        
      </Text>
    </View>
    <View
      style={
        Object {
          "alignSelf": "stretch",
          "flex": 1,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(0, 0, 0, 0.87)",
          }
        }>
        10.04.2017 12:52
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "rgba(0, 0, 0, 0.54)",
          }
        }>
        Köln
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Section Section with Custom Title Color 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "flex": 0,
        },
        Object {
          "borderBottomColor": "rgba(0, 0, 0, 0.12)",
          "borderBottomWidth": 1,
        },
      ]
    }>
    <View
      style={
        Object {
          "flex": 0,
          "height": 48,
          "justifyContent": "center",
          "paddingLeft": 16,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "red",
            },
            Object {
              "fontSize": 14,
              "fontWeight": "bold",
            },
          ]
        }>
        Foo bar
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
          },
          false,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Section Section with Title 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "flex": 0,
        },
        Object {
          "borderBottomColor": "rgba(0, 0, 0, 0.12)",
          "borderBottomWidth": 1,
        },
      ]
    }>
    <View
      style={
        Object {
          "flex": 0,
          "height": 48,
          "justifyContent": "center",
          "paddingLeft": 16,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "rgba(0, 0, 0, 0.54)",
            },
            Object {
              "fontSize": 14,
              "fontWeight": "bold",
            },
          ]
        }>
        Foo bar
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
          },
          false,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Section Section without Border 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "flex": 0,
        },
        false,
      ]
    }>
    <View
      style={
        Object {
          "flex": 0,
          "height": 48,
          "justifyContent": "center",
          "paddingLeft": 16,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "rgba(0, 0, 0, 0.54)",
            },
            Object {
              "fontSize": 14,
              "fontWeight": "bold",
            },
          ]
        }>
        Foo bar
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
          },
          false,
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
      </Text>
    </View>
  </View>
</View>
`;

exports[`Storyshots Section Section without Title 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E0E0E0",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "flex": 0,
        },
        Object {
          "borderBottomColor": "rgba(0, 0, 0, 0.12)",
          "borderBottomWidth": 1,
        },
      ]
    }>
    <View
      style={
        Array [
          Object {
            "paddingBottom": 16,
            "paddingLeft": 16,
            "paddingRight": 16,
          },
          Object {
            "paddingTop": 16,
          },
        ]
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail">
        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
      </Text>
    </View>
  </View>
</View>
`;
